### ISSUE TITLE
We Need To Revive This Project With A Fork

### ISSUE THREAD

**Issue by Pedroviski01** [InoriRus/Kyty#106]
The creator of this project hasn't been online since december 22th 2022, i gonna try to fork this project, anyone to help me?

**Comment by JustAStickmin** [comment]
what coding language is it in?

**Comment by raleighlittles** [comment]
Would love to help out with a fork if you have established a list of work items that you need done! Or is there a discord/forum already setup to discuss what the next steps are?

**Comment by archonveins** [comment]
this is really sad bc i cant get my issues resolved. I wanted to play but i get errors none got a solution for.
Let alone reviving the project...

and yeah im posting in 2025

**Comment by CONDUCTOR77747** [comment]
Неужели автор ушел на СВО в декабре 2022?((((

**Comment by Spincial** [comment]
This would be interesting to see.

**Comment by ilan0098** [comment]
https://github.com/shadps4-emu/shadPS4
This one seems active
Also, I think it might use the same way to emulate

**Comment by SplaatSites** [comment]
@ilan0098 Yes, but it will probably not emulate PS5 games, but there is an emulator for that: [sharpemu](https://github.com/par274/sharpemu).

**Comment by Nmzik** [comment]
> Неужели автор ушел на СВО в декабре 2022?((((

Или СВО или может предложили ему хорошую работу где-то?

**Comment by KnextKoder** [comment]
Bruh, idk but I think the og creator of this repo is dead, most likely a casualty of the Russia Ukraine war from forced conscription.
Last commit was on Oct 3rd 2022.
If true... RIP🕊
